rskj-core/src/main/java/co/rsk/validators/BlockTimeStampValidationRule.java

```diff
@@ -46,8 +46,22 @@ public boolean isValid(Block block) {
         boolean result = blockTime - currentTime <= this.validPeriodLength;
```

Comment:
isnt `blockTime - currentTime` always negative? Then `result` will always be true!

Reply:
Is the spirit of this validation giving some room for a block in the future? It's very unclear from the code above, and we would benefit from comments.

Reply:
A clock may be a bit out of sync. We just don't want blocks in the future. This can be used to lower the difficulty for example.